uni/lib/view/schedule/schedule.dart

```diff
@@ -57,7 +66,10 @@ class SchedulePageViewState extends State<SchedulePageView>
       vsync: this,
       length: 6,
     );
-    final weekDay = DateTime.now().weekday;
+
+    currentWeek = Week(start: widget.now);
+
+    final weekDay = widget.now.weekday;
```

Comment:
I need the `now` here because I need its weekday to calculate the index of the tab that should be open. Does this mean you want me to have both `now` and `currentWeek` in the `SchedulePageView`?

Reply:
Yes, that'd be fine

Reply:
Done!